api/cookie.go

```diff
@@ -36,8 +36,8 @@ var (
 // writeCookie is a helper function that writes the given JWT token as a
 // secure cookie.
 func writeCookie(w http.ResponseWriter, token string, exp int64) error {
-	if exp <= 0 || time.Unix(exp, 0).Before(time.Now()) {
-		exp = time.Now().Unix()
+	if exp <= 0 || time.Unix(exp, 0).Before(time.Now().UTC()) {
+		exp = time.Now().UTC().Unix()
```

Comment:
not sure why we care the expiry is before the current time if we're going to set it to the current time anyway? those would immediately expire right?

Reply:
Hm, you're right. I'm not sure what I was thinking back then. I'll remove the check.